### PR TITLE
Added PWA manifest for express-2021 lesson 1

### PR DESCRIPTION
**Pull request by daynew**
This PR adds a `manifest.json` file to Lesson 1 of Express-2021, as foundational work for the offline PWA pilot. The file will only be loaded if the user's browser cookie `offline_pilot` is set to true, so this code is not currently active for any users in production.

The `offline_pilot` browser cookie was implemented by adding a Rails route at `/offline/join_pilot`, at which the browser sets the `offline_pilot` cookie to `true` and redirects the user back to the dashboard home page.
## Links
Jira: https://codedotorg.atlassian.net/browse/FND-1762?atlOrigin=eyJpIjoiYzg1YTQzNmJkOWY0NDE0MTk0ZjBhNTJkNmYyYjc3YjgiLCJwIjoiaiJ9 

## Testing story
We manually verified that the browser cookie in Chrome is set to true when visiting localhost-studio.code.org:3000/offline/join_pilot 

## Follow-up work
The browser cookie solution is not necessarily the final decision for our experiment flag implementation. However, for now it is sufficient to ensure this added code does not immediately trigger manifest downloads after merging.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -26,6 +26,7 @@
     -# making use of the dcdo feature.
     %script{data: {'dcdo': DCDO.frontend_config.to_json}}
     -# webpack-runtime.js must appear exactly once on any page containing webpack entries
+
```
Comment (daynew) [inline dashboard/app/views/layouts/application.html.haml]:
```suggestion
```

File: dashboard/app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -36,6 +37,9 @@
     = javascript_include_tag 'https://cdn.optimizely.com/js/12977480133.js' if Rails.env.production?
     %script{src: webpack_asset_path('js/code-studio-common.js')}
     = csrf_meta_tags
+    -# Load the manifest.json for Maze if user is in the offline pilot
```
Comment (daynew) [inline dashboard/app/views/layouts/application.html.haml]:
```suggestion
    -# Load the manifest.webmanifest for Maze if user is in the offline pilot
```

File: dashboard/public/s/express-2021/lessons/1/manifest.webmanifest [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Code.org Express 2021",
```
Comment (daynew) [inline dashboard/public/s/express-2021/lessons/1/manifest.webmanifest:2]: Lets add a JIRA item to make `manifest.webmanifest` translatable.

**Review by annaxuphoto — APPROVED** [review]
this is dayne's approval of anna's code but dayne owns the PR and cannot approve his own PR :)

**Pull request merged**